src/crypto/client/NUT12.ts

```diff
@@ -14,11 +13,11 @@ function arraysEqual(arr1: Uint8Array, arr2: Uint8Array) {
 
 export const verifyDLEQProof = (
 	dleq: DLEQ,
-	B_: ProjPointType<bigint>,
-	C_: ProjPointType<bigint>,
-	A: ProjPointType<bigint>,
+	B_: WeierstrassPoint<bigint>,
+	C_: WeierstrassPoint<bigint>,
+	A: WeierstrassPoint<bigint>,
 ) => {
-	const sG = secp256k1.ProjectivePoint.fromPrivateKey(bytesToHex(dleq.s));
+	const sG = secp256k1.Point.BASE.multiply(bytesToNumber(dleq.s));
```

Comment:
If I read [the documentation](https://github.com/paulmillr/noble-curves) correctly, I think this should probably be:

```suggestion
	const sG = secp256k1.Point.BASE.multiply(secp256k1.Point.Fn.fromBytes(dleq.s));
```

Reply:
On reflection, I think you are correct here, as the point multiplication reduces modulo `n`.

Reply:
I have made this into an issue  #433. also have a specific PR to attach to it of we decide to fix it across the code base there are several files that could benefit from this clean up